apps/site/pages/en/blog/community/2025-06-28-Emelia-Smith.md

```diff
@@ -0,0 +1,37 @@
+---
+date: '2025-06-28T13:00:00.000Z'
```

Comment:
FYI to whoever lands this–please update this before land

Reply:
Good for me!